script/openqa-powermanagement.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python3
+
+# TODO:
+#  * Add return values checks to avoid crashes/traces (no connection to openQA server, etc.)
+#  * The host name of machines may not be unique
+
+import configparser
+import argparse
+import json
+import requests
+import subprocess
+
+machine_list_idle = []
+machine_list_offline = []
+machine_list_busy= []
+machines_to_power_on = []
+
+jobs_worker_classes = []
+
+config_file = "/etc/openqa/openqa.ini"
```

Comment:
```suggestion
config_file = os.environ.get("OPENQA_CONFIG", "/etc/openqa") + "/openqa.ini"
```

Reply:
Great, thanks for the suggestion!

Reply:
Done.

Reply:
I'm wondering what's the benefit of that. All OSs I know (including Windows) should support `/` and multiple consecutive `/` are not a problem at all as well.